schema.graphql

```diff
@@ -993,6 +993,8 @@ input UpdateOrganizationInput {
 }
 
 input UpdateUserInput {
+  id: ID
+  applangcode : String
```

Comment:
This key should be camelCase, `appLanguageCode`, make this change wherever necessary

Reply:
Done